src/EnergyPlus/DXCoils.cc

```diff
@@ -16838,7 +16840,12 @@ void CalcVRFCoolingCoil_FluidTCtrl(EnergyPlusData &state,
             ShowFatalError(state, format("{} \"{}\" - Rated total cooling capacity is zero or less.", thisDXCoil.DXCoilType, thisDXCoil.Name));
         }
 
-        TotCap = thisDXCoil.RatedTotCap(Mode);
+        if (present(MaxCoolCap)) {
+            TotCap = min(MaxCoolCap, thisDXCoil.RatedTotCap(Mode));
+        } else {
+            TotCap = thisDXCoil.RatedTotCap(Mode);
+        }
+
```

Comment:
Could this be replaced as `TotCap = min(MaxCoolCap, thisDXCoil.RatedTotCap(Mode));` and just pass in the limit? or default it and only change this one line.

Reply:
I see, leave it as-is.

Reply:
Okay